lib/Doctrine/Common/Cache/PhpFileCache.php

```diff
@@ -42,14 +42,13 @@ public function __construct($directory, $extension = self::EXTENSION)
      */
     protected function doFetch($id)
     {
-        $filename = $this->getFilename($id);
+        // note: error suppression is still faster than `file_exists`, `is_file` and `is_readable`
+        $value = @include $this->getFilename($id);
```

Comment:
The error suppression should cover only the inclusion, not the filename computation IMO